src/Bicep.Core/Parsing/ExpressionFlags.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Extensions;
+using Bicep.Core.Navigation;
+using Bicep.Core.Syntax;
+
+namespace Bicep.Core.Parsing
+{
+    [Flags]
+    public enum ExpressionFlags
+    {
+        None = 0,
+        AllowComplexLiterals = 1,
+        AllowResourceDeclarations = 2,
+        InsideColonDelimitedContext = 4,
```

Comment:
>InsideColonDelimitedContext [](start = 8, length = 27)

Can you add a comment on `InsideColonDelimitedContext`?

Reply:
Adding